ruby/Ice/filesystem/client.rb

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env ruby
+# Copyright (c) ZeroC, Inc.
+
+require 'etc'
+require 'Ice'
+
+# Load the code generated by the Slice compiler. The Slice module Filesystem maps to a Ruby module with the same name.
+require_relative 'Filesystem.rb'
+
+
+# Recursively prints the contents directory "dir" in tree fashion.
+# For files, show the contents of each file.
+#
+# @param [FileSystem::DirectoryPrx] dir The directory to list.
+# @param [Integer] depth The current depth in the directory hierarchy.
+def listRecursive(dir, depth = 0)
```

Comment:
```suggestion
def listRecursive(dir, depth)
```

We always pass depth explicitly, no need for this default.